src/pybind/mgr/dashboard/frontend/src/app/shared/api/user.service.ts

```diff
@@ -0,0 +1,32 @@
+import { HttpClient } from '@angular/common/http';
+import { Injectable } from '@angular/core';
+
+import { UserFormModel } from '../../core/auth/user-form/user-form.model';
+import { ApiModule } from './api.module';
+
+@Injectable({
+  providedIn: ApiModule
+})
+export class UserService {
+  constructor(private http: HttpClient) {}
+
+  list() {
+    return this.http.get('api/user');
+  }
+
+  delete(username: string) {
+    return this.http.delete(`api/user/${username}`);
+  }
+
+  get(username: string) {
+    return this.http.get(`api/user/${username}`);
+  }
+
+  create(user: UserFormModel) {
```

Comment:
As we have API tests for all other services it would be kind to maintain that even so it's not very fancy ;)